zerver/lib/rate_limiter.py

```diff
@@ -67,15 +90,17 @@ def remove_ratelimit_rule(range_seconds, num_requests):
 def block_user(user, seconds, domain='all'):
```

Comment:
For this one, just taking out the "user" probably leaves it too terse, as `block`. (Hard to search for that name, for example.) `block_access` would be pretty good.